lib/modules/powershell/code_execution/invoke_assembly.py

```diff
@@ -0,0 +1,142 @@
+from __future__ import print_function
+
+from builtins import str
+from builtins import object
+from lib.common import helpers
+import base64
+
+
+class Module(object):
+
+    def __init__(self, mainMenu, params=[]):
+
+        self.info = {
+            'Name': 'Invoke-Assembly',
+
+            'Author': ['@kevin'],
+
+            'Description': "Loads the specified assembly into memory and invokes the main method. "
+                           "The Main method and class containing Main must both be PUBLIC for "
+                           "Invoke-Assembly to execute it",
+
+            'Software': '',
+
+            'Techniques': ['T1059'],
+
+            'Background': True,
+
+            'OutputExtension': None,
+
+            'NeedsAdmin': False,
+
+            'OpsecSafe': True,
+
+            'Language': 'powershell',
+
+            'MinLanguageVersion': '2',
+
+            'Comments': [
+                'Assemblies are loaded with reflection into the current process. This method is',
+                'different than Cobalt Strike\'s execute-assembly as it does not create a new process',
+                'or inject any code via WriteProcessMemory',
+            ]
+        }
+
+        self.options = {
+            'Agent': {
+                'Description': 'Agent to run module on.',
+                'Required': True,
+                'Value': ''
+            },
+            'Assembly': {
+                'Description': 'Local path to the .NET assembly (.exe)',
```

Comment:
Maybe add something like "This can be an absolute or relative path." Just so it's clear.